File: superhero/src/main/java/com/frontguys/superhero/models/Response.java
package com.frontguys.superhero.models;

import java.util.Date;

public class Response {
    private int id;
    private int requestId;
    private int contractorId;
    private Date date;
    private ClientDetails contractorDetails;
    private Request request;
    private int payment;
    private Date plannedDate;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getRequestId() {
        return requestId;
    }

    public void setRequestId(int requestId) {
        this.requestId = requestId;
    }

    public int getContractorId() {
        return contractorId;
    }

    public void setContractorId(int contractorId) {
        this.contractorId = contractorId;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public Request getRequest() {
        return request;
    }

    public void setRequest(Request request) {
        this.request = request;
    }

    public int getPayment() {
        return payment;
    }

    public void setPayment(int payment) {
        this.payment = payment;
    }

    public Date getPlannedDate() {
        return plannedDate;
    }

    public void setPlannedDate(Date plannedDate) {
        this.plannedDate = plannedDate;
    }

    public ClientDetails getContractorDetails() {
        return contractorDetails;
    }

    public void setContractorDetails(ClientDetails contractorDetails) {
        this.contractorDetails = contractorDetails;
    }
}

File: superhero_flutter/lib/models/response.dart
import 'package:superhero_flutter/models/client_details.dart';
import 'package:superhero_flutter/models/request.dart';

class Response {
  int id;
  int requestId;
  int contractorId;
  ClientDetails contractorDetails;
  String date;
  Request request;
  int payment;
  String plannedDate;

  Response.fromJson(map) {
    id = map['id'];
    requestId = map['requestId'];
    contractorId = map['contractorId'];
    contractorDetails = ClientDetails.fromJson(map['contractorDetails']);
    date = map['date'];
    request = Request.fromJson(map['request']);
    payment = map['payment'];
    plannedDate = map['plannedDate'];
  }
}
